webtau-rest-groovy/src/test/groovy/com/twosigma/webtau/http/HttpTest.groovy

```diff
@@ -274,6 +276,61 @@ class HttpTest {
         assert varArgHeader == mapBasedHeader
     }
 
+    @Test
+    void "no body request"() {
+        def noBodyExpectation = {
+            body.should == ""
```

Comment:
Should it?  I would have thought no body should make body be null, not empty string.

Reply:
Oh, I see.  This is really an artifact of how `TestServerResponseEcho` is implemented.  You can ignore this question :)